docs/overview/learning-path-beginner.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Start Here! A guide for beginners making Office Add-ins
+description:  A recommended path for beginners through the learning resources for Office Add-ins.
+ms.date: 02/19/2020
+ms.topic: conceptual
+ms.custom: scenarios:getting-started
+localization_priority: Priority
+---
+
```

Comment:
Thought: Link to this page from the "Get started" section of the ["Office Add-ins documentation" page](https://review.docs.microsoft.com/en-us/office/dev/add-ins/?branch=beginner-learning-path)?

Reply:
Done